src/libmongoc/src/mongoc/mcd-integer.h

```diff
@@ -0,0 +1,380 @@
+#ifndef MCD_INTEGER_H_INCLUDED
+#define MCD_INTEGER_H_INCLUDED
+
+#include <mongoc-prelude.h>
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
This does not seem particularly necessary. Remove?

Reply:
Practically unecessary because everything is `static` and therefore bounded to the TU. This just becomes a habit when creating C headers.